README.md

```diff
@@ -106,7 +106,7 @@ Before each test run, clean up the repo:
 
 To run tests locally its a simple case of calling the command below from this directory
```

Comment:
s/its/it's or it is

I realize this isn't your change, but as long as you're changing the file we might as well fix errors we find. We should probably add a colon to the end of that statement as well.